chart/docs/mcp-inspector-azure-app-routing-deployment.md

```diff
@@ -0,0 +1,187 @@
+# Azure App Routing Deployment Guide
```

Comment:
I think we need a doc `docs/helm-chart.md` for external users

Reply:
we have /aks-mcp/chart/README.md